hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -230,8 +230,7 @@ private void rollOver() throws IOException {
   }
 
   private void createNewFile() throws IOException {
-    this.output =
-        fs.create(this.logFile.getPath(), false, bufferSize, replication, WriterBuilder.DEFAULT_SIZE_THRESHOLD, null);
+    this.output = fs.create(this.logFile.getPath(), false, bufferSize, replication, HoodieCommonConfig.LOG_FILE_BLOCK_SIZE.defaultValue(), null);
```

Comment:
Should the actual configured value be passed in, instead of the default value?

Reply:
thanks， you are right。 modified